<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WangDaKa</class>
 <widget class="QWidget" name="WangDaKa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>忘打卡申请</string>
   </property>
  </widget>
  <widget class="closeButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>0</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_wdk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>851</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>忘打卡申请</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓  名(*)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>部    门：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>填表日期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wdkStartTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_wdkStart">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> 职    务：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_position">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>忘打卡日期：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>199</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>忘打卡原因(*)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_wdkReason">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>721</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dealTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_wdkStartTime">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>248</width>
      <height>197</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_department">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_wdkType">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>如果换休，请备注换休日期</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBox_sb">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>上班忘打卡</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_xb">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下班忘打卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_wdkEndTime">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>248</width>
      <height>197</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>忘打卡时段(*)：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>副总经理处理</string>
   </property>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>部门领导处理</string>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>证明人处理</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_allLeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>191</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>审核人</string>
   </property>
   <widget class="QComboBox" name="comboBox_leader1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>证明人(*)：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_leader3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>副总经理(*)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>考勤员(*)：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_leader4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>部门领导(*)：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_leader2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>考勤员处理</string>
   </property>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_OK
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pushButton_OK:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pushButton_OK:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_Cancel
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
}

QPushButton#pushButton_Cancel:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton#pushButton_Cancel:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>closeButton</class>
   <extends>QToolButton</extends>
   <header location="global">closebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
